roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -79,6 +79,23 @@
     user: "{{ ansible_user_id }}"
     key: "{{ pub_ssh_key['content'] | b64decode }}"
 
+- name: Prepare the static IP address for controller.
+  when:
+    - _layout.vms.controller.ip_address is defined
+  vars:
+    _ip_address: >-
+      {{
+        _layout.vms.controller.ip_address.address
+      }}
+    _ip_gw: "{{ _layout.vms.controller.ip_address.gw }}"
+    _ip_dns: "{{ _layout.vms.controller.ip_address.dns }}"
+  ansible.builtin.template:
+    src: templates/static_ip.j2
+    dest: "/tmp/controller_0_static_ip.sh"
```

Comment:
maybe we should expose this as an artifacts instead? For the sake of later debugging? Especially thinking "CI" here.

Reply:
Thank you. Yes it should be there.